Description: fix the script to allow user to provide options 
Author: anantshri
Bug-kali: https://bugs.kali.org/view.php?id=2940#c4396
Last-Update: 2015-12-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/DirBuster-1.0-RC1.sh
+++ b/DirBuster-1.0-RC1.sh
@@ -1,3 +1,3 @@
 #!/bin/bash
-java -Xmx256M -jar /usr/share/dirbuster/DirBuster-1.0-RC1.jar
+java -Xmx256M -jar /usr/share/dirbuster/DirBuster-1.0-RC1.jar "$@"
 
